apps/web/src/components/devtools/toolbar.tsx

```
@@ -1,18 +1,47 @@
+
+  if (hideMode === "section") {
+    return null;
+  }
+
+  if (hideMode === "temporary") {
+    return (
+      <button
+        onClick={handleShowToolbar}
+        className="fixed bottom-0 left-1/2 -translate-x-1/2 mb-2 px-3 py-1 bg-background-secondary border border-border rounded text-xs font-mono hover:bg-background-tertiary transition-colors outline-none focus-visible:ring-1 focus-visible:ring-ring z-50"
+        aria-label="Show toolbar"
+      >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)